### DISCUSSION TITLE
Can't deploy properly on Vercel

### DISCUSSION THREAD

**Discussion by bachiloglu** [vercel/community#5501] [Help]
### Summary

I'm using Astro to build a little portfolio. I uploaded it without problems to GitHub and also made a `npm run build` that worked locally.

I also tried to deploy to Netlify and works properly.

That's what I get when deploy to Vercel (the deploy works, but I got this error):

![image](https://github.com/vercel/community/assets/24626759/be5de7a6-4520-4265-9d4a-b2e995fc410e)

Here is my astro.config.mjs:

```
import { defineConfig } from 'astro/config';
import tailwind from "@astrojs/tailwind";

import vercel from "@astrojs/vercel/serverless";

// https://astro.build/config
export default defineConfig({
  integrations: [tailwind()],
  output: "server",
  adapter: vercel()
});
```

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/24626759/4b8e7446-ded7-4f03-885c-708b0e86536a)

https://porfolio-navy-sigma-58.vercel.app/

**Comment by rafaelcacilhas** [comment]
Im having a similar issue. This [branch](https://github.com/rafaelcacilhas/astroship) contains the Astroship template with the only modification being the inclusion of

```
import vercel from "@astrojs/vercel/serverless";
export default defineConfig({
  output: 'server',
  adapter: vercel()
}); 
```

on astro.config.mjs. Trying to deploy it on vercel  returns an `error 500`
![image](https://github.com/vercel/community/assets/100581315/aada1951-3321-44c6-9973-b16436399d8e)

The only logs on vercel are `TypeError: Cannot read properties of undefined (reading 'exec')`
![image](https://github.com/vercel/community/assets/100581315/92bac3de-45ae-45eb-8e7c-067dfb9cdde8)

My previous attempt has a different log on vercel, but seems like a similar issue 
```
TypeError: Cannot read properties of undefined (reading 'endsWith')
    at removeTrailingForwardSlash (file:///var/task/vercel/path0/.vercel/output/_functions/chunks/astro-assets-services_d72f8614.mjs:10:15)
```
